scripts/Linux-Pkg/Makefile

```diff
@@ -49,7 +49,16 @@ build: pack-source
 	rm -rf Output/pkg
 	rm -rf Output/src
 
+aur:
```

Comment:
Normally, what we do is build all the stuff in a Docker container (as you can see, there's a `docker-build` directive in Makefile), so we don't need to install dependencies (or ever Arch) on the physical host. 
We can have two "Docker directives", one that builds the unsigned pkg, and one that builds and deploys the code to AUR. To achieve that, we need to copy the local `.ssh` folder into the container, so it can push the package. 

We do a similar thing into the Debian Makefile, you can check it to see how we do it.